strapi/strapi
# Adding Croatian (Hrvatski) language for Admin & Upload plugin

### What does it do?

This PR just adds Croatian language that I needed in my project and though that other might also.

### Why is it needed?

Because my client has users that do not know English that well, translation to Croatian was a requirement.
Until now, I had a patch-package (was running on 4.12.5) but now this is not the way anymore, so, here is a PR to make this part of the Strapi.

### How to test it?

Simply add "hr" into locales setup in app.js and it should be available as admin user language (selectable).

### Related issue(s)/PR(s)

None
